packages/hub/routes/config.ts

```diff
@@ -0,0 +1,34 @@
+import Koa from 'koa';
+import autoBind from 'auto-bind';
+import { JSONAPIDocument } from '../utils/jsonapi-document';
+import config from 'config';
+
+export const DEGRADED_THRESHOLD = 10;
+
+export default class ConfigRoute {
+  constructor() {
+    autoBind(this);
```

Comment:
Would it be good to have a check at initialization that the config for essential urls exist, and if not, report to Sentry? Or maybe a Checkly check?

Reply:
Possible future enhancement